rfcs/text/0042-risk-score-extensions.md

```diff
@@ -24,16 +21,50 @@ This RFC seeks to extend the [existing risk fields](https://www.elastic.co/guide
     * These categories' definitions may be _extended_ in the future to include new data sources
   * Due to the above category traits, we need to come up with a naming convention for these categorical score fields that allows them to be extended without invalidating the existing field names.
 
-<!--
-Stage 1: If the changes include field additions or modifications, please create a folder titled as the RFC number under rfcs/text/. This will be where proposed schema changes as standalone YAML files or extended example mappings and larger source documents will go as the RFC is iterated upon.
--->
 
 <!--
 Stage X: Provide a brief explanation of why the proposal is being marked as abandoned. This is useful context for anyone revisiting this proposal or considering similar changes later on.
 -->
 
 ## Fields
 
+I'll enumerate the fields being introduced here grouped by their motivation/goal:
+
+### Identifier Fields
+These fields are intended to allow future extensibility of our concept of an "identifier." Currently, we leverage two such fields: `host.name` and `user.name`, to group documents for the purposes of risk scoring, but we definitely imagine this being extended in the future to allow multiple such fields for hosts and users, or more simply to allow administrators to configure fields other than those above. The presence of these two fields allows us to audit/explain how a particular risk score was identified.
+
+* `identifier_field`
+* `identifier_value`
+
+
+### Risk Category Fields
+Some of the context here was discussed in Stage 0; please read the above for that. More specifically, these fields seek to provide the category contributions to the score, and the number of risk inputs in that category, across each of the five proposed categories:
```

Comment:
One other note (which I can add to the RFC, but let's discuss here): The main motivation for having separate fields here per category (as opposed to an `object` like in `riskiest_inputs`) is the ability to search/filter scores on these attributes.

Reply:
Point being: if there's a better way to do this other than `type: nested` I'm all ears 😄